config/orm-services.yml

```diff
@@ -56,7 +63,7 @@ services:
     knp.doctrine_behaviors.translatable_subscriber.default_locale_callable:
         class:           "%knp.doctrine_behaviors.translatable_subscriber.default_locale_callable.class%"
         arguments:
-            - "%locale%"
+            - %locale%
```

Comment:
Parameters should still be wrapped under quotes for YAML best practices and to avoid deprecations